docs/crocks/Reader.md

```diff
@@ -0,0 +1,61 @@
+# Reader
+
+`Reader e a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Reader` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`ask`](#ask), [`of`](#of), [`type`](#type) |  [`ap`](#ap), [`chain`](#chain), [`inspect`](#inspect), [`map`](#map), [`of`](#of), [`run`](#runWith), [`type`](#type) |
+
+## Constructor
+
+### ask
+
+`Reader m => m e ~> a -> m e a`
```

Comment:
and `ask` takes a function that will receive the environment
(check the source code to see how `fn` is used)